docs/source/flow-project.rst

```diff
@@ -109,7 +169,7 @@ We can then execute this workflow with:
     hello 2b985fa90138327bef586f9ad87fc310
     # ...
 
-If we implemented and integrated the operation and condition functions correctly, calling the ``run`` command twice should produce no output the second time, since the ``greeted()`` condition is met for all jobs and the ``hello()`` operation should therefore not be executed.
+If we implemented and integrated the operation and condition functions correctly, calling the ``run`` command twice should produce no output the second time, since the ``greeted`` condition is met for all jobs and the ``hello`` operation should therefore not be executed.
```

Comment:
Calling `run` again might print progress output or something like that, but we can promise that it will "take no action" or "execute no operations."
```suggestion
If we implemented and integrated the operation and condition functions correctly, calling the ``run`` command twice should execute no operations the second time, since the ``greeted`` condition is met for all jobs and the ``hello`` operation should therefore not be executed.
```